config.yaml

```diff
@@ -10,8 +10,7 @@ disableKinds:
 - taxonomyTerm
 
 params:
-  description: |
-    Jaeger, inspired by [Dapper](https://research.google.com/pubs/pub36356.html) and [OpenZipkin](https://zipkin.io/), is a distributed tracing system released as open source by Uber Technologies. It can be used for monitoring microservice-based architectures.
+  tagline: Monitor and troubleshoot transactions in distributed systems
```

Comment:
I like the new version (`Monitor....`) more than the old one (`... inspired by...`), but how about something like: 

> Open Source distributed tracing system created to monitor cloud native applications and microservices architectures

Credit goes to @JStickler and @objectiser  :)

Reply:
I think the difference between the Title and the Tagline is that the Title is "what is it" (end-to-end distributed tracing) and the Tagline is "what is it for". So let's take this apart:
  * "distributed tracing system" - redundant as we already say it in the title
  * "open source" - maybe, but we could also say that in the following section, not in the tagline (cf. https://kubernetes.io/)
  * "microservices architectures" - a misnomer, microservices are not an architecture, they are a deployment choice. The same architecture can be implemented as a monolith or as microservices
  * technically Jaeger can even be used to profile monoliths (in fact that's how tracing started at Uber), but I an not sure we need to mention that
  * both "cloud native" and "microservices" serve more as buzz words than an accurate description of the target. "complex distributed systems" would be more accurate. But buzz words might be good for PR

Possible alternative (two lines): 
```
Monitor and troubleshoot transactions in complex distributed systems,
cloud native and microservices-based application
```

Reply:
PS - I like the one-liner better

Reply:
"complex distributed systems" is pretty much a synonym for "microservices" nowadays :) 

Not relevant for this specific PR, but I would be interested in hearing more details about "microservices architecture" being a misnomer, as I have accepted Martin Fowler's definition so far, where he says that it's an "architectural style".

Reply:
> "complex distributed systems" is pretty much a synonym for "microservices" nowadays :)

yes and no. Do you consider Cassandra a microservices-based solution? I think most people wouldn't, but it's certainly falls under the umbrella of a "complex distributed system".

> more details about "microservices architecture" being a misnomer

I think on a high level "architecture" refers to logical components that work together as a single application. More than 10yrs ago, when nobody has used the term "microservices", we used to architect a system as a collection of services (SOA), each implementing a certain API and performing a certain business function. Then at deployment time we could decide whether we want to deploy several services in a single JVM (and let them talk via direct API calls) or in multiple JVMs (and let them talk via SOAP). Or take today's example - Jaeger at Uber. Our collector certainly qualifies as a microservice, yet it contains several business "services". Deciding whether a given service runs as a separate process or is running in a larger process that combines multiple services is a deployment question, not an architectural question.

Anyway, this is just an opinion.